kernel/include/paging.h

```diff
@@ -2,6 +2,33 @@
 //https://os.phil-opp.com/page-tables/  x64 4 layer paging
 //P4 P3 P2 P1 represent the different pagetable layers
 
+/*
+    This is currently set up to provide convenient way of creating pagetables, 
+    mapping and contstraining memory usage, which is a necessity for processes. 
+    Paging, as in swapping pages between disk and RAM will come later. 
```

Comment:
I don't know if putting a page on disk while it's not being used is called "Paging". It's also not required, you put pages on disk when you run out of RAM because loading a page from disk into ram on context switches is veeeeery slow.